docs/source/install.rst

```diff
@@ -32,6 +32,60 @@ Install from Source
     $ pip install git+https://github.com/pydata/pandas-gbq.git
 
 
+Install in Development Mode
+---------------------------
+
+Follow these instructions to install pandas-gbq if you want to make changes to
+the library.
+
+`Download the code <https://github.com/pydata/pandas-gbq/archive/master.zip>`__
+from GitHub or clone the git repository.
+
+.. code-block:: shell
+
+    $ git clone https://github.com/pydata/pandas-gbq.git
+    $ cd pandas-gbq
+
+Pip
+~~~
+
+To install pandas-gbq and its dependencies in development mode via pip
+
+.. code-block:: shell
+
+    $ pip install -e .
+
+Conda
+~~~~~
+
+To install packages in development mode using conda, first
+install the conda build tools package in your root conda environment.
+
+.. code-block:: shell
+
+    $ conda install conda-build
+
+Optionally, create a new conda environment.
+
+.. code-block:: shell
+
+   $ conda create --no-deps -n pandas-gbq
+   $ source activate pandas-gbq
+
+Install pandas-gbq in development mode
+
+.. code-block:: shell
+
+    $ conda develop .
+
+Install the necessary dependencies
+
+.. code-block:: shell
+
+    $ conda install --channel conda-forge \
```

Comment:
so the point of this is you *don't* have to specify secondary deps, they will simply be installed via the dependency graph. it looks like you install

``conda create -n foo --dry google-auth-oauthlib google-api-python-client pandas google-auth-httplib2``

though I am suprised that ``google-auth`` is actually a dep of the ``google-auth-oauthlib`` (and httplib2), you usually don't do it this way, rather installing ``google-auth`` should install all libs under its tree, so this is a bit odd.